script/foreman-debug

```diff
@@ -303,6 +354,42 @@ add_files /etc/sysconfig/pgsql
 add_files "/var/lib/pgsql/data/pg_log/*"
 add_cmd "foreman-rake plugin:list" "plugin_list"
 
+if [ $DB_DUMP -eq 1 ]; then
+  if [ "$DB_ADAPTER" = "postgresql" ]; then
+    printv "Making full PostgreSQL database dump"
+    su postgres -c "pg_dump -Fc $DB_NAME" > "$DIR/database_pg.dump"
```

Comment:
Use the existing db:dump rake tasks for this, no need for detecting adapters etc.